src/components/table/README.md

```diff
@@ -576,6 +576,8 @@ values: `sm`, `md`, `lg`, or `xl`.
   clips off any content that goes beyond the bottom or top edges of the table. In particular, this
   may clip off dropdown menus and other third-party widgets.
 
+- Sometime Responsive will not work!. We can add table-class="table-responsive text-nowrap" in < b-table >.
+
```

Comment:
It is better to explain why `fixed` and `responsive` to not work well together. this solution doesn't walways work, specifically when the table is narrower than the available width:

![image](https://user-images.githubusercontent.com/2781561/73143293-d4dc2980-406e-11ea-9d2b-7e63ec419e88.png)